toolchain/check/member_access.cpp

```diff
@@ -546,12 +546,13 @@ auto PerformCompoundMemberAccess(
   return member_id;
 }
 
-auto PerformTupleIndex(Context& context, SemIR::LocId loc_id,
-                       SemIR::InstId tuple_inst_id, SemIR::InstId index_inst_id)
-    -> SemIR::InstId {
+auto PerformTupleAccess(Context& context, SemIR::LocId loc_id,
+                        SemIR::InstId tuple_inst_id,
+                        SemIR::InstId index_inst_id) -> SemIR::InstId {
   tuple_inst_id = ConvertToValueOrRefExpr(context, tuple_inst_id);
   auto tuple_inst = context.insts().Get(tuple_inst_id);
   auto tuple_type_id = tuple_inst.type_id();
+  auto tuple_index = SemIR::ElementIndex::Invalid;
```

Comment:
Does this need to be declared up here, or could we declare it down on line 594 where it's initialized?